x-pack/solutions/chat/plugins/workchat-app/public/application/pages/integrations.tsx

```diff
@@ -7,11 +7,15 @@
 
 import React from 'react';
 import { useBreadcrumb } from '../hooks/use_breadcrumbs';
-import { useIntegrationList } from '../hooks/use_integration_list';
-import { IntegrationListView } from '../components/integrations/listing/integration_list_view';
+import { IntegrationActiveView } from '../components/integrations/listing/integration_active_view';
+import { IntegrationCatalogView } from '../components/integrations/listing/integration_catalog_view';
 
-export const WorkChatIntegrationsPage: React.FC<{}> = () => {
+export const WorkChatIntegrationsPage: React.FC = () => {
   useBreadcrumb([{ text: 'Integrations' }]);
-  const { integrations } = useIntegrationList();
-  return <IntegrationListView integrations={integrations} />;
+  return <IntegrationActiveView />;
+};
+
+export const WorkChatCatalogPage: React.FC = () => {
+  useBreadcrumb([{ text: 'Integrations' }]);
+  return <IntegrationCatalogView />;
```

Comment:
NIT: We ideally want one `Page` per file. Why? Because even if we don't do it today, it allows to do lazy/suspense page loading later (only loading the components used by a page when the page is displayed, instead of loading all the components of the whole app when on first app rendering)